lumps/dehacked/dehacked.txt

```diff
@@ -610,54 +610,54 @@ BGFLAT30 = SLIME13
 BGFLAT15 = AQF004
 BGFLAT31 = AQF021
 # Obituary texts, for ZDoom-based ports.
```

Comment:
Can we replace this line with a comment so someone doesn't undo this fix later?
```
# Obituary texts for sourceports.
# NOTE: Boom-based DEHACKED parsers like those in PrBoom+ and Woof! require a strict
# "a = b" pattern *including the spaces*. Do not remove them!
```

Reply:
Actually, forget it, everything else already follows this format and I'm not even sure why the obituaries don't. It's fine without the comment.